versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+     serviceaccount/mongodb-enterprise-ops-manager configured
+     Warning: resource roles/mongodb-enterprise-appdb is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
+     role.rbac.authorization.k8s.io/mongodb-enterprise-appdb configured
+     Warning: resource rolebindings/mongodb-enterprise-appdb is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
+     rolebinding.rbac.authorization.k8s.io/mongodb-enterprise-appdb configured
+     WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     serviceaccount/mongodb-enterprise-appdb created
+     serviceaccount/mongodb-enterprise-database-pods created
+     serviceaccount/mongodb-enterprise-ops-manager created
+     role.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     rolebinding.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/richie/mongo-setup/kubeconfig/merged.config
+     serviceaccount/mongodb-enterprise-appdb created
+     serviceaccount/mongodb-enterprise-database-pods created
+     serviceaccount/mongodb-enterprise-ops-manager created
+     role.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     rolebinding.rbac.authorization.k8s.io/mongodb-enterprise-appdb created
+     ```
+
+13. Switch to the master cluster and generate Ops Manager API keys and add our IP addresses to the Ops Manager access list using the following steps:
+
+   1. Get the Ops Manager URL using the following command:
+
+     ```
+    URL=http://$(kubectl -n "mongodb-operator" get svc ops-manager-svc-ext -o jsonpath='{.status.loadBalancer.ingress[0].ip}:{.spec.ports[0].port}') echo $URL
+    ```
+    
+   2. On the master cluster, use the following command to generate Ops Manager API keys and add IP addresses to the Ops Manager access list:
+
+     :::note
+     The below command will only work if your service is exposed as a LoadBalancer, incase it is exposed as nodeport service then kindly 
```

Comment:
The command below works only ... 

in case, it is  or if it is 

Use present tense over future tense